src/borderarray.jl

```diff
@@ -91,7 +91,7 @@ end
 
 @inline function _outeraxes(arr, border)
     map(axes(arr), border.lo, border.hi) do r, lo, hi
-        (first(r)-lo):(last(r)+hi)
+        Base.IdentityUnitRange((first(r)-lo):(last(r)+hi))
```

Comment:
I'm not familiar with the code base thus I don't know if this change is expected.

But to keep compatibility to julia v1.0, you could add a custom implementation
```julia
if VERSION < v"1.1"
    IdentityUnitRange = Base.IdentityUnitRange
else
    IdentityUnitRange = # self-defined
end
```

Reply:
The best portability option is `OffsetArrays.IdentityUnitRange` which is defined here: https://github.com/JuliaArrays/OffsetArrays.jl/blob/e22f56ed7f80048ebe9662eeae7dc2891ecf0b8d/src/OffsetArrays.jl#L6-L10